### ISSUE TITLE
Update?

### ISSUE THREAD

**Issue by TheDude53** [pakastin/open-source-flash#129]
It's almost February of 2020. Is this letter still going to be sent? I think there is still an opportunity to send it and make a change. Just wanted to check in on things.

**Comment by pakastin** [comment]
Yeah, we’ve tried to contact Adobe. I think they can’t just go and open source yet, since that would open attacks. Plus there’s licensed parts, which they probably can’t never open source.

So I guess we’ll need to wait until support ends. If you have any other ideas we’re all ears!

**Comment by Brian151** [comment]
honestly, who knows when/if all that third-party stuff would get opened-up...
that's still no excuse for not opening the source. They'd just have to exclude the third-party crap. [and that wouldn't really stop the open source community from reverse-engineering the proprietary bits]

large chunks of the SWF [and FLV] format are not documented either because they aren't licensed for that, or it's beyond the scope of the specs.

what kinds of attacks? hacks? those are already a problem...XD adobe is most likely scared of someone trying to de-throne the IDE more than anything else. It was initially promised the XFL format would be open-source. It's still not. It is ...somewhat self-documenting, but that's about it. Even so, replacing or even CHALLENGING an IDE like flash/animate, Unity, etc... ? No small feat... A command line tool or a library is so much easier than a full-fledged editor.

**Comment by MZPIGGEE81** [comment]
[https://github.com/open-source-flash/open-source-flash/issues/129#issue-555985531](https://github.com/open-source-flash/open-source-flash/issues/129#issue-555985531) 
Duplicate of #

**Comment by MZPIGGEE81** [comment]
@MZPIGGEE81

**Comment by TheDude53** [comment]
I know #138 was opened and all, but any official updates?
